Modules/System/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -86,14 +88,30 @@ codeunit 9999 "Upgrade Tag"
         UpgradeTagImpl.SetAllUpgradeTags(NewCompanyName);
     end;
 
+    /// <summary>
+    /// With this method you get all the upgrade tags by company in a list. 
+    /// </summary>
+    procedure GetPerCompanyUpgradeTags(var PerCompanyUpgradeTags: List of [Code[250]])
```

Comment:
Double check that all the signatures list the parameters.

Reply:
Add the comment for the parameter. Hope this is what you mean